content/en/faq/security/verify-mtls-encryption.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How can I verify Istio mTLS indeed encrypt the traffic?
+weight: 40
+---
+
+Once you turned on the mutual TLS, traffic is encrypted. We will show that through `tcpdump` in the
+sidecar container.
+
+To enable tcpdump, you need to install Istio with proxy priviledged enabled.
+
+{{< text bash >}}
+$ istioctl manifest apply --set values.global.proxy.privileged=true
+{{< /text >}}
+
+{{< warning >}}
+This installation is only for demo purpose, please revert the change after you are done for better security!
+{{< /warning>}}
+
+Deploy httpbin, sleep workloads, and a sleep-legacy deployment which does not have sidecar injected.
+
+{{< text bash >}}
+$ kubectl label ns default istio-injection=enabled
+$ kubectl apply -f ./samples/httpbin/httpbin.yaml
+$ kubectl apply -f ./samples/sleep/sleep.yaml
+$ kubectl apply -f - <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sleep-legacy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sleep-legacy
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app: sleep-legacy
+    spec:
+      containers:
+      - name: sleep
+        image: governmentpaas/curl-ssl
+        command: ["/bin/sleep", "3650d"]
+        imagePullPolicy: IfNotPresent
+EOF
+serviceaccount/httpbin unchanged
+service/httpbin created
+deployment.apps/httpbin created
+serviceaccount/sleep unchanged
+service/sleep created
+deployment.apps/sleep created
+deployment.apps/sleep-legacy created
+{{< /text >}}
+
+Configuring peer authentication as `STRICT`.
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
+metadata:
+  name: "default"
+spec:
+  mtls:
+    mode: STRICT
+EOF
+peerauthentication.security.istio.io/default created
+{{< /text >}}
+
+Now httpbin is protected under mutual TLS. We execute tcpdump in httpbin pod sidecar container.
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod  -lapp=sleep -ojsonpath={.items..metadata.name}) -c istio-proxy -it -- sudo tcpdump dst port 80  -A
```

Comment:
Does this assume tcpdump is always on the proxy image?

Reply:
i think so. i'm using 1.5.0 build. distroless might be a bit different